mmv1/templates/terraform/examples/firestore_index_datastore_mode.tf.erb

```diff
@@ -0,0 +1,20 @@
+resource "google_firestore_index" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
We need to register this example in `Index.yaml` to be able to generate test and doc. See step 5 at https://googlecloudplatform.github.io/magic-modules/develop/test/#add-a-create-test

Reply:
Done. Thanks for the pointer!